src/core_plugins/kibana/public/visualize/listing/create_or_delete_button.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { CreateButtonLink } from 'ui_framework/components/button/create_button_link';
+import { DeleteButton } from 'ui_framework/components/button/delete_button';
+
+export function CreateOrDeleteButton({ showCreate, doDelete }) {
+  if (showCreate) {
+    return <CreateButtonLink href = "#/visualize/new"
+                             tooltip = "Create new visualization" />;
```

Comment:
If we put the closing bracket on its own line, we'll be aligned with the HTML styleguide:

```javascript
return (
  <CreateButtonLink
    href = "#/visualize/new"
    tooltip = "Create new visualization"
  />
);
```